[DX-1860] add deprecation warning (#6075)
TykTechnologies/tyk-docs: tyk-docs/content/tyk-portal-api.md

  importance: 0
---

{{< warning success >}}

> **Attention: Upcoming deprecation for Tyk Classic portal**

Reviewer: ```suggestion
**Attention: Future deprecation of Tyk Classic Portal**
```

> You’ve reached a page related to the *Tyk Classic Developer Portal*. This product will soon be discontinued. If you

Reviewer: ```suggestion
You’ve reached a page related to the *Tyk Classic Portal*. This product is no longer actively developed as it has been superseded by the new [Tyk Enterprise Developer Portal]({{< ref "portal/overview" >}}).
```

> haven’t started using it, please use our new [Developer Portal]({{< ref "portal/overview" >}}) instead.

Reviewer: ```suggestion
```

> This page is for the classic developer portal, which has been superseded by our latest portal. Please note that the

Reviewer: ```suggestion
```

> classic developer portal has limited support and maintenance. Please contact us at 

Reviewer: ```suggestion
Please note that the Tyk Classic Portal now has limited support and maintenance. Please contact us at 
```

> [support@tyk.io](<mailto:support@tyk.io?subject=Tyk classic developer portal>)if you have any questions.

Reviewer: ```suggestion
[support@tyk.io](<mailto:support@tyk.io?subject=Tyk Classic Portal>) if you have any questions.
```
